src/frontend/shortlist/src/components/preferences/PreferenceDemo.vue

```diff
@@ -44,6 +47,36 @@ export default {
   },
   setup() {
     return {
+      question1: {
+        Name: "Instruction",
+        Text: "How much do you value instructions from your teachers?",
+        Min: 1,
+        MinLabel: "I'd like more time studying alone.",
+        Max: 7,
+        MaxLabel: "I wish instructors can cover every details",
+      },
+      question2: {
+        Name: "Supportive Environment",
+        Text: "How much do you weight the importance of a supportive environment?",
+        Min: 1,
+        MinLabel: "I'm a loner for life.",
+        Max: 7,
+        MaxLabel: "I want a family like school environment!",
+      },
+      question3: {
+        Name: "Borough",
+        Text: "Is there a spicific borough you are looking for?",
+        Options: ["Manhattan", "Brooklyn", "Queens", "Bronx"],
+      },
+      question4: {
+        Name: "Academic Performance",
+        Text: "How would you rank your academic performance so far?",
+        Min: 1,
+        MinLabel: "Average",
+        Max: 7,
```

Comment:
I like the questions. The majority of questions are slider-type. Would it be possible to paraphrase them and change the type accordingly? Or Find a few different types of questions from dataset. I am concerned about type because they will also reflect on the profile and the user will experience kind of the same type set of questions.

Reply:
I removed all question that require type in input, this restriction will make it easier for user to select and easier to control the result for calculation of algorithm, drop down and slide makes the best sense to me